### DISCUSSION TITLE
Maximum number of team capacity?  How best to scale teams/bot?

### DISCUSSION THREAD

**Discussion by amino916** [coccoinomane/crabada.py#100] [General]
1. Does the bot have a theoretical limit to teams?  Does it have a recommended / practical limit?  

What team counts are you using and is it effective?

_The goal is not to sacrifice much mining efficiency..._

2. What is the best way to deploy additional teams along with the bot?  How to best add wallet address(es)?

**Comment by coccoinomane** [comment]
Hello @amino916!

A user has reported to use the bot with 58 teams, more details here > https://github.com/coccoinomane/crabada.py/issues/107

Cheers,
Cocco
